src/constant_texts.c

```diff
@@ -101,6 +102,9 @@ const char *ui_text_options_logging_export[] = {
     "Enable logs",
 };
 
+const char *ui_text_card_pairing_success = "All cards are paired";
+const char *ui_text_card_pairing_warning = "Pair all cards for proper use";
```

Comment:
any reference used for the change in text?

Reply:
I have discussed the changes with Rohan.. added reference on clickup ticket